README.md

```diff
@@ -10,4 +10,16 @@ Example:
     <div bind-html-compile="feincmsPages.pageSlug.regions.main"></div>
     <div bind-html-compile="feincmsPages.pageSlug.regions.sidebar"></div>
 </div>
-```
\ No newline at end of file
+```
+
+## Releasing a new version
+
+1. Commit your changes.
+1. Run `grunt build` to generate the compiled template files
+1. Follow the guidelines at http://semver.org/ to determine your new version number.
+1. Update `CHANGELOG.md` with your new version number and a description of changes.
+1. Update the `version` property in `bower.json` and `package.json`
+1. Commit those changes with the commit message "Bump to [version number]". [version number] should be in the format x.y.z.
```

Comment:
`[version number] should be in the format x.y.z.`
->
`[version number] should be the same semver value used in package.json`